com.unity.render-pipelines.core/Documentation~/View-Lighting-Tool.md

```diff
@@ -1,28 +1,17 @@
-## View Lighting tool
+## Light Anchor
 
-The View Lighting tool allows you to set up lighting in camera space. It uses the main Camera (the one that renders to the Game view) to set the orientation of the Light using spherical coordinates, and it uses a target and distance to set the position of the Light.
+The Light Anchor can help to place light sources around subjects, in relation to a Camera and an anchor point. It's particularly effective for cinematic lighting, which often requires multiple light sources orbiting a subject.
 
-Since the View Lighting tool uses camera-space, it is especially useful when setting up lighting for cinematics.
+## Using the Light Anchor Component
 
-To be able to use the view lighting tool, we need to have at least one camera tagged as "MainCamera" (only one will be used).
+To use the Light Anchor, you must set the Tag of at least one Camera to "MainCamera". By default, the Anchor's position will be the same as the position of the GameObject the Light Anchor Component is attached to.
 
-## Using the View Lighting tool
+Use the **Orbit** and **Elevation** to control the orientation of the light, in degrees, relative to the main Camera's and Anchor's positions. Use the **Distance** to control how far from the anchor, in meters, you want to place the Light.
```

Comment:
Right after "**Use the **Orbit** and **Elevation** to control the orientation of the light, in degrees, relative to the main Camera's and Anchor's positions.**", we also need to add:

"If the Light has a Cookie or an IES Profile, use the **Rolll** to change their orientation."